<launch>

	<node name="copter_control" pkg="copter_control" type="copter_control" output="screen" launch-prefix="xterm -e">

		<!-- Pilot Command rate -->
		<param name="rate" value="30.0"/>

		<param name="enable_xy_stab" value="true"/>
		
		<!-- max pithc\roll errors -->
		<param name="max_pitch" value="3.0"/>
		<param name="max_roll" value="3.0"/> 
		<param name="max_xy_vel" value="0.8"/>
		<param name="max_z_vel" value="0.2"/> <!--0.2-->
		<param name="min_z_vel" value="-0.2"/> 

		<!-- XY command (pithc\roll) PID gains -->
		<param name="xy_p_gain" value="4.0"/>
		<param name="xy_i_gain" value="1.0"/> 
		<param name="xy_d_gain" value="0.05"/> 
		<param name="xy_i_max" value="1.0"/>
		<param name="xy_i_min" value="-1.0"/> 

		<!-- XY velocity goal PID gains -->
		<param name="xy_vel_goal_p_gain" value="1.0"/> 
		<param name="xy_vel_goal_i_gain" value="0.06"/>  
		<param name="xy_vel_goal_d_gain" value="1.2"/> 
		<param name="xy_vel_goal_i_max" value="0.1"/>
		<param name="xy_vel_goal_i_min" value="-0.1"/>

		<!-- Z command PID gains -->
		<param name="z_p_gain" value="1.3"/> <!--2.0-->
		<param name="z_i_gain" value="0.013"/> <!--1.2-->
		<param name="z_d_gain" value="0.0"/> <!--0.6-->
		<param name="z_i_max" value="0.5"/> <!--0.03-->
		<param name="z_i_min" value="-0.5"/> 

	</node>


</launch>

